src/utils.ts

```diff
@@ -281,19 +292,20 @@ export async function getVp(
         space,
         delegation
       }
-    })
+    }
   };
 
   try {
-    const res = await fetch(options.url, init);
-    const json = await res.json();
-    if (json.error) return Promise.reject(json.error);
-    if (json.result) return json.result;
```

Comment:
AFAIK, no 200 response should ever return an object with the `error` key

Reply:
> Also, consider that the getVp function returned the result if body.result otherwise it returned undefined. It can be an error from the past, but better to check

What do you mean by that ?

Reply:
I mean [after this line](https://github.com/snapshot-labs/snapshot.js/pull/904/files#diff-39b2554fd18da165b59a6351b1aafff3714e2a80c1435f2de9706355b4d32351L291) it returns nothing(`undefined`). But your implementation always returns data. As I said, it could be an error in the previous version, but better to check in places where we use `getVp`

Reply:
I think you're reading the wrong commit, the line you're mentioning have been removed